src/platform/forms-system/src/js/utilities/validations/index.js

```diff
@@ -55,6 +58,11 @@ export function isValidPartialDate(day, month, year) {
   return true;
 }
 
+export function isValidDate(day, month, year) {
```

Comment:
We can just return this directly
 `export function isValidDate(day, month, year) {
  return moment({ day, month: parseInt(month, 10) - 1, year }).isValid();
}`

and we're going for the high score we can do it again for the linting warning on line 53 if you want
`export function isValidPartialDate(day, month, year) {
  return !(year && !isValidYear(year));
}`

All tests still pass